src/modules/create-market/components/create-market-form-outcome/create-market-form-outcome.jsx

```diff
@@ -209,6 +210,11 @@ export default class CreateMarketOutcome extends Component {
       updatedMarket.tickSize = value
     }
 
+    // Always check if (maxPrice - minPrice) / precision is an even number
+    if (((scalarBigNum - scalarSmallNum) / numTicksBigNum) % 2 !== 0) {
```

Comment:
This isn't safe for BigNumbers

Reply:
I was originally using `(scalarBigNum.sub(scalarSmallNum).div(numTicksBigNum)).mod(2) !== 0` in the switch statement above, which wasn't working. (I realize now I should have been using `minus` instead.) I had temporarily switched to using regular math operators just to see if moving it below the switch statement would work the way I wanted, but forgot to switch it back to using normal big number operators. Will update.

Reply:
Actually doing `console.log(typeof scalarBigNum)` outputs `string` here, so I'll need to make sure all of these values are BigNumbers at this point in the code.

Reply:
Ah, the issue is that `scalarBigNum` is the empty string when the page first loads, so it's not getting set to a BigNumber up above. I'll add a check to make sure all of the numbers are actually BigNumbers.